nbsite/pyodide/__init__.py

```diff
@@ -129,8 +129,8 @@ def extract_extensions(code: str) -> List[str]:
         js_modules.update(design_resources['js_modules'])
     resources = Resources(mode='cdn')
     extensions = _bundle_extensions(None, resources)
-    js += [bundle.cdn_url for bundle in extensions if bundle.cdn_url and
-           '@holoviz/panel@' not in bundle.cdn_url]
+    js += [cdn_url for bundle in extensions if bundle.cdn_url and
+           '@holoviz/panel@' not in (cdn_url:= str(bundle.cdn_url))]
```

Comment:
Is this just to annoy @jlstevens ? :D

Reply:
I see you @Hoxbro

Reply:
I would love to take thecredit for annoying @jlstevens, but this one goes to @philippjfr.

Reply:
It's a sect!

Reply:
<img src="https://github.com/holoviz-dev/nbsite/assets/19758978/8dd13016-82cd-46a9-9e92-6f658c1ba5f9" alt="image" width="500" height="500">